Description: Fixes conflicting array sizes for variable `TabCom' in
  TabCommodules/FvwmScript/FvwmScript.c and modules/FvwmScript/Instructions.c
  Upstream did not want patch.
Author: Michael Tautschnig <mt@debian.org>
Forwarded: not-needed
Debian-Bug: http://bugs.debian.org/801612
Reviewd-By: Jaimos Skriletz <jaimosskriletz@gmail.com>
Last-Updated: 2015-10-12
--- a/modules/FvwmScript/FvwmScript.c
+++ b/modules/FvwmScript/FvwmScript.c
@@ -78,7 +78,7 @@ Window ref;
 FlocaleWinString *FwinString;
 
 extern int yyparse(void);
-extern void (*TabCom[25]) (int NbArg,long *TabArg);
+extern void (*TabCom[29]) (int NbArg,long *TabArg);
 
 Display *dpy;
 int screen;
